### DISCUSSION TITLE
Cannot start service worker

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#3028] [Help with using Postal]
Hello all,

Running postal 3.2.2, Docker version 27.0.3, and Ubuntu 22.04.4.  I started getting the below error today when doing postal start
`Removing postal_web_1
Removing postal_smtp_1
Removing postal_worker_1
Recreating 44e73454dbb2_postal_web_1 ... 
Recreating f3bc3625019f_postal_worker_1 ... error
Recreating fd3001b5454f_postal_smtp_1   ... 
Recreating 44e73454dbb2_postal_web_1    ... error
ERROR: for f3bc3625019f_postal_worker_1  Cannot start service worker: error while creating mount source path '/opt/postal/config': mkdir /opt/postal: read-only file system

ERROR: for 44e73454dbb2_postal_web_1  Cannot start service web: error while creating mount source pRecreating fd3001b5454f_postal_smtp_1   ... error

ERROR: for fd3001b5454f_postal_smtp_1  Cannot start service smtp: error while creating mount source path '/opt/postal/config': mkdir /opt/postal: read-only file system

ERROR: for worker  Cannot start service worker: error while creating mount source path '/opt/postal/config': mkdir /opt/postal: read-only file system

ERROR: for web  Cannot start service web: error while creating mount source path '/opt/postal/config': mkdir /opt/postal: read-only file system

ERROR: for smtp  Cannot start service smtp: error while creating mount source path '/opt/postal/config': mkdir /opt/postal: read-only file system
ERROR: Encountered errors while bringing up the project.`

**Comment by willpower232** [comment]
Was Postal working well and then stopped working?

If it was working before, has anyone changed the permissions?

Presumably you aren't doing anything special with your /opt directory?

Is it possible your server rebooted poorly and is having a terrible time? Might be worth another reboot to see if it can boot cleanly.

**Reply by jshapiro2** [reply]
It was working correctly before.  Could not figure out so just setup a new one on the latest version.
